vm/vm_test.go

```diff
@@ -456,7 +465,7 @@ for {
 `,
 		`
 a = []
-for i = 0; i < 10000; i++ {
+for i = 0; i < 100000; i++ {
```

Comment:
please separate changes for TestInterrupts

Reply:
Reverted TestInterrupts count change